data/tools/pywmlx/state/machine.py

```diff
@@ -13,7 +13,14 @@
 import pywmlx.nodemanip
 import pdb
 
+# Universe - convenient singleton for which
```

Comment:
It looks like some stuff from PR #7540 leaked into this one.

Reply:
No, this is intended.